php/class-wp-cli.php

```diff
@@ -1233,6 +1255,185 @@ private function get_posts_with_missing_terms( $author_taxonomy, $post_types = [
 		// phpcs:enable
 	}
 
+	/**
+	 * This function handles obtaining an author account which should have the most posts assigned to it. If unable
+	 * to find an appropriate account, this function will attempt to create an author account for use.
+	 *
+	 * @return WP_User
+	 * @throws Exception If unable to successfully create an author user account.
+	 */
+	public function get_most_prolific_author() {
```

Comment:
This is a very opinionated approach.. It makes sense.. but I think creating an author with a random display name doesn't.

What do you think of simply assigning them to the admin instead?

Reply:
My initial thought with going with this approach was obfuscation. Going with an admin account as you're suggesting avoids creating random user accounts though, and seems like a cleaner solution, so I've updated the code to use that approach instead.

Reply:
My mind was changed once again, this time by @naxoc 's suggestion that we use postmeta markers to indicate whether a post should be skipped if the `post_author` doesn't exist. This is a better approach, as it would leave things as they are, allowing web admins to figure out which User ID no longer exists and investigate as to why.